tf2_ros/src/buffer.cpp

```diff
@@ -107,10 +100,10 @@ void Buffer::onTimeJump(const struct rcl_time_jump_t & time_jump)
   if (RCL_ROS_TIME_ACTIVATED == time_jump.clock_change ||
     RCL_ROS_TIME_DEACTIVATED == time_jump.clock_change)
   {
-    ROS_WARN("Detected time source change. Clearing TF buffer.");
+    RCLCPP_WARN(node_->get_logger(), "Detected time source change. Clearing TF buffer.");
```

Comment:
This is a bug. We're assuming that `node_` is not `nullptr`, but in fact that is the default value. The node argument to `tf2_ros::Buffer` is optional.

Reply:
Fixed in https://github.com/ros2/geometry2/pull/393